src/js/utils/UserSettings.js

```diff
@@ -39,7 +40,8 @@
         maintainRatio : true,
         resizeContent : false,
         origin : 'TOPLEFT'
-      }
+      },
+      COLOR_FORMAT: 0,
```

Comment:
Oh yeah of course, silly me. That way I can skip the constant entirely. Thanks, I'll fix it tomorrow.